packages/trip-details/src/types.ts

```diff
@@ -13,14 +13,31 @@ export interface DepartureDetailsProps {
   departureDate: Date;
 }
 
+export enum FareTableText {
+  regular = "regular",
+  youth = "youth",
+  senior = "senior",
+  special = "special",
+  cash = "cash",
+  electronic = "electronic"
+}
+
+export interface FareTableLayout {
+  cols: {
+    header: FareTableText;
+    key: string;
+  }[];
+  header: FareTableText;
+}
 export interface TransitFareData {
   [key: string]: Money
 }
 
 export interface FareDetailsProps {
```

Comment:
Since we're now displaying the fare by leg table in the dropdown rather than under the question mark, we could make a new type and revert this one back to original so that we don't need a breaking change (by the way, is there any easy way to see if we ever committed a breaking change?). Should we keep max/min TNC fare info available for the fare details or is that outdated now anyway?

Reply:
Yes, you should revert the breaking change. Leave the min/max TNC fare (it's no relevant to this PR).